package.json

```diff
@@ -33,6 +33,7 @@
     "@nestjs/passport": "^7.1.5",
     "@nestjs/platform-express": "^7.6.13",
     "@nestjs/swagger": "^4.8.0",
+    "axios": "^0.21.1",
```

Comment:
I dont think we need axios as a new dependency here. We can use the `Nest-Http-Module` like [here](https://github.com/EddieHubCommunity/api/blob/eb559302ebb3da5d7344a9309b0c18d7e769bbbd/src/github/geocoding.service.ts#L48)

Reply:
oh ok, I am kinda new to nestjs. will change this.